test/offline/test_cql.py

```diff
@@ -68,15 +68,43 @@ def get_args():
     return parser.parse_known_args()[0]
 
 
-def test_cql(args=get_args()):
+def add_returns(buffer: ReplayBuffer, gamma: float = 0.99) -> ReplayBuffer:
+    data_dict = buffer._meta.__dict__
+    start_idx = np.concatenate([np.array([0]), np.where(data_dict["done"])[0] + 1])
+    end_idx = np.concatenate(
+        [np.where(data_dict["done"])[0] + 1, np.array([len(data_dict["done"])])],
+    )
+    ep_rew = [data_dict["rew"][i:j] for i, j in zip(start_idx, end_idx, strict=True)]
+    ep_ret = []
+    for i in range(len(ep_rew)):
+        episode_rewards = ep_rew[i]
+        disc_returns = [0] * len(episode_rewards)
+        discounted_return = 0
+        for j in range(1, len(episode_rewards) + 1):
+            discounted_return = (
+                episode_rewards[len(episode_rewards) - j] + gamma * discounted_return
+            )
+            disc_returns[len(episode_rewards) - j] = discounted_return
+        ep_ret.append(disc_returns)
+
+    new_data_dict = data_dict.copy()
+    ep_rets = np.concatenate(ep_ret)
+    new_data_dict["calibration_returns"] = ep_rets
+    new_batch = Batch(**new_data_dict)
+    buffer._meta = new_batch
+    return buffer
+
+
+def test_cql(args=get_args(), calibrated=False):
     if os.path.exists(args.load_buffer_name) and os.path.isfile(args.load_buffer_name):
         if args.load_buffer_name.endswith(".hdf5"):
-            buffer = VectorReplayBuffer.load_hdf5(args.load_buffer_name)
+            buffer = ReplayBuffer.load_hdf5(args.load_buffer_name)
```

Comment:
Why was that needed?

Reply:
It's the same in fact, `VectorReplayBuffer` just inherits `load_hdf5` from `ReplayBuffer`. And having `VectorReplayBuffer.load_hdf5' is kind of misleading that buffer should be a vector buffer.

Reply:
It's not the same, since load_hdf5 is a classmethod and despite being inherited, it would instantiate a VectorReplayBuffer and not a ReplayBuffer. The type is not annotated sufficiently well to express it, I will open a mini PR now to fix it

Reply:
I think for offline we don't need `VectorReplayBuffer` since (at least as understand it) it is meant to work with `VectorEnv` but in offline we don't have an `Env`. But anyway for that code both work fine without a problem.

Reply:
Resolving. FYI, the type issue is fixed with #942

Reply:
Unresolved this, pls change back to VectorReplayBuffer. Even if ReplayBuffer makes sense here, should happen in a separate PR